quanttrader/strategy/strategy_base.py

```diff
@@ -113,13 +113,11 @@ def adjust_position(self, sym, size_from, size_to, timestamp=None):
         o.full_symbol = sym
         o.order_type = OrderType.MARKET
         o.order_size = size_to - size_from
-        o.source = self.id  # identify source
-        if timestamp is None:
-            o.create_time = datetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')
-        else:
+        if timestamp is not None:
```

Comment:
Shouldn't we stamp an order if it does not have one?

Reply:
It's is already done in the callee, i.e. place_order (). Hence no need to do that in adjust_position ()

Reply:
I was adding `adjust_postion` as a shortcut to place market order. It does have some redundency checks.